Dockerfile

```
@@ -1,12 +1,12 @@
-FROM fedora:38 as builder
+FROM debian:11 as builder
 ARG TARGETARCH
-RUN dnf install clang llvm make libbpf-devel -y
+RUN apt-get update && apt-get install -y curl clang gcc llvm make libbpf-dev -y
 RUN curl -LO https://go.dev/dl/go1.20.linux-${TARGETARCH}.tar.gz && tar -C /usr/local -xzf go*.linux-${TARGETARCH}.tar.gz
 ENV PATH="/usr/local/go/bin:${PATH}"
 WORKDIR /app
 COPY . .
 RUN make build
 
-FROM registry.fedoraproject.org/fedora-minimal:38
+FROM gcr.io/distroless/base@sha256:73deaaf6a207c1a33850257ba74e0f196bc418636cada9943a03d7abea980d6d
```

[> ### Base Operating System
> 
> Distroless images are based on Debian 11 (bullseye). Images are explicitly tagged with Debian version suffixes (e.g. `-debian11`). Specifying an image without the distribution will currently select `-debian11` images, but that will change in the future to a newer version of Debian. It can be useful to reference the distribution explicitly, to prevent breaking builds when the next Debian version is released.](https://github.com/GoogleContainerTools/distroless/tree/main#base-operating-system)

To avoid breaking changes, let's pin the underlying distro.

```suggestion
FROM gcr.io/distroless/base-debian11@sha256:73deaaf6a207c1a33850257ba74e0f196bc418636cada9943a03d7abea980d6d
```